pom.xml

```diff
@@ -616,6 +616,12 @@
                 <version>4.5.12</version>
             </dependency>
             
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpmime</artifactId>
+                <version>4.5.12</version>
```

Comment:
We should consider (as a separate task) update to current versions of httpcomponents.